Content.Server/_DV/CosmicCult/CosmicCultRuleSystem.cs

```diff
@@ -0,0 +1,709 @@
+using Content.Server._DV.CosmicCult.Components;
+using Content.Server._DV.CosmicCult.EntitySystems;
+using Content.Server.Actions;
+using Content.Server.Administration.Logs;
+using Content.Server.Antag;
+using Content.Server.Atmos.Components;
+using Content.Server.Audio;
+using Content.Server.Bible.Components;
+using Content.Server.Chat.Systems;
+using Content.Server.CrewManifest;
+using Content.Server.EUI;
+using Content.Server.GameTicking.Events;
+using Content.Server.GameTicking.Rules;
```

Comment:
Removed unused using directives

```suggestion
using Content.Server._DV.CosmicCult.Components;
using Content.Server.Actions;
using Content.Server.Administration.Logs;
using Content.Server.Antag;
using Content.Server.Atmos.Components;
using Content.Server.Audio;
using Content.Server.Bible.Components;
using Content.Server.Chat.Systems;
using Content.Server.EUI;
using Content.Server.GameTicking.Rules;
```